projects/ng-imgix/src/lib/imgix.component.spec.ts

```diff
@@ -166,9 +166,9 @@ describe('Imgix Component', () => {
 
   describe('type guards', () => {
     const TEST_CASES: [
-      html: string,
-      attribute: string,
-      expectedValue: string,
+      /* html:  */string,
+      /* attribute:  */string,
+      /* expectedValue:  */string,
```

Comment:
Just checking, but we can't include these fields anymore?

Reply:
Yeah. These were added in Typescript 4.0, but I had to downgrade to Typescript 3.9 to be in the supported range for Angular v10. I commented them out so we can just uncomment them in the future when we can upgrade Typescript.